File: src/TaskList/wwwroot/appScripts/Task.js
"use strict";
class Task {
    constructor() {
        this.Id = 0;
        this.Name = '';
        this.IsDeleted = false;
        this.DateCreated = '';
        this.ListId = 0;
        this.IsCompleted = false;
        this.taskIndex = 0;
        this.listIndex = 0;
    }
}
exports.Task = Task;

File: src/TaskList/Models/Task.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TaskList.Models
{
    public class Task
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public DateTime DateCreated { get; set; }
        public bool IsDeleted { get; set; }
        public int ListId { get; set; }
        public bool IsCompleted { get; set; }
    }
}
